plugin/wavesurfer.regions.js

```diff
@@ -60,12 +67,17 @@ WaveSurfer.Regions = {
             }
 
             var duration = my.wavesurfer.getDuration();
+            if (typeof e.targetTouches !== 'undefined' && e.targetTouches.length === 1) {
```

Comment:
Here we could also check if the id of the ended touch is the same as the started touch, but I think it's good enough for now.